Description: fix the use of the oldest package when multiple packages exist.
             (Closes: #841207)
Author: Michael Bonhomme <spike.sp@free.fr>
Last-Update: 2016-10-18
Index: apt-rdepends-1.3.0/apt-rdepends
===================================================================
--- apt-rdepends-1.3.0.orig/apt-rdepends
+++ apt-rdepends-1.3.0/apt-rdepends
@@ -256,7 +256,7 @@ sub get_depends {
   }
   else {
     if (my $i = $p->{VersionList}) {
-      if (my $j = pop(@$i)) {
+      if (my $j = shift(@$i)) {
 	return $j->{DependsList};
       }
     }
